### ISSUE TITLE
.NET 6 Web API not working without .NET Core 2.1 Runtime in the server.

### ISSUE THREAD

**Issue by ponkumarpandian** [dotnet/runtime#69784]
We are using below nuget packages for our WebAPI project. Since .NET Core 2.1 support has been end, as per our organization policy we are suppose to remove the .NET Core 2.1 runtime from the server as this is consider as a vulnerability. But we uninstall the .NET core 2.1 version , application not working and getting 500 internal server error .

Project 'XXX.YYY.ABC.FFF.Service' has the following package references
   [net6.0]:
   Top-level Package                                    Requested   Resolved
   > Microsoft.AspNetCore.Authentication.JwtBearer      6.0.0       6.0.0
   > Microsoft.AspNetCore.Cors                          2.2.0       2.2.0
   > Microsoft.AspNetCore.Mvc.NewtonsoftJson            6.0.0       6.0.0
   > Microsoft.EntityFrameworkCore                      6.0.0       6.0.0
   > Microsoft.EntityFrameworkCore.Design               6.0.0       6.0.0
   > Microsoft.EntityFrameworkCore.Relational           6.0.0       6.0.0
   > Microsoft.EntityFrameworkCore.SqlServer            6.0.0       6.0.0
   > Microsoft.IdentityModel.Tokens                     6.14.1      6.14.1
   > ncrontab                                           3.3.1       3.3.1
   > Serilog.Extensions.Logging.File                    2.0.0       2.0.0
   > System.DirectoryServices.Protocols                 6.0.0       6.0.0
   > System.Linq                                        4.3.0       4.3.0
   > System.Linq.Expressions                            4.3.0       4.3.0

**Comment by teo-tsirpanis** [comment]
The CORS package is an old version and does not seem to be required. Have you tried removing it and running the application again?

Same with the two packages at the bottom. If you only target .NET 6 you don't need them.

**Comment by ponkumarpandian** [comment]
yes, I am checking, how about the "Microsoft.IdentityModel.Tokens" it has dependencies with .NET Standard 2.0?
